src/EditorFeatures/CSharpTest/Classification/SemanticClassifierTests.cs

```diff
@@ -2325,5 +2325,118 @@ void Main(string s)
     }
 }", Keyword("var"));
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.Classification)]
+        public async Task TestUnmanagedConstraint_InsideMethod()
+        {
+            await TestInMethodAsync(
+                "var unmanaged = 0;",
+                Keyword("var"));
```

Comment:
❓ Does this automatically verify that `unmanaged` is *not* marked as a keyword here?

Reply:
Yes. There should've been another entry if it was a keyword. I'll add a comment.

---
In reply to: [171661262](https://github.com/dotnet/roslyn/pull/25033#discussion_r171661262) [](ancestors = 171661262)